git-archive-all.sh

```diff
@@ -178,7 +178,7 @@ while test $# -gt 0; do
 done
 
 OLD_PWD="`pwd`"
-TMPDIR=${TMPDIR:-/tmp}
+TMPDIR=/tmp
```

Comment:
Why hardcode `/tmp`? The whole point of a `$TMPDIR` variable is to let the OS provide a tmp directory. On Mac OS X, `echo $TMPDIR` produces a directory inside `/var/folders`, not `/tmp`.